checker/bin/checkIntegration

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
this file name `chckIntegration` is not consistent with the other bash file?

Reply:
Shall I rename it to alluxio-checker.sh ? 
And alluxio-check.sh will invoke spark-checker.sh, mapreduce-checker.sh....